src/main/java/racingcar/domain/CarName.java

```diff
@@ -0,0 +1,27 @@
+package racingcar.domain;
+
+import racingcar.exception.InvalidCarNameException;
+
+public class CarName {
+    private static final int MAX_NAME_LENGTH = 5;
+    private final String carName;
+
+    public CarName(String carName) {
+        validCarNameLength(carName);
+        this.carName = carName;
+    }
+
+    public String getCarName() {
+        return carName;
+    }
+
+    private void validCarNameLength(String carsName) {
+        if (carsName.length() > MAX_NAME_LENGTH) {
```

Comment:
```suggestion
        if (carsName.length() > MAX_NAME_LENGTH) {
             throw new InvalidCarNameException();
        }
```
만 되어도 충분할 것 같습니다 😄 
이미 `throw`를 하고 있어서 `catch`가 굳이 필요해 보이지 않습니다 😄